geopy/geocoders/base.py

```diff
@@ -70,6 +70,8 @@ class options(object):
             If specified, tunnel requests through the specified proxy.
             E.g., ``{"https": "192.0.2.0"}``. For more information, see
             documentation on :class:`urllib.request.ProxyHandler`.
+            Can be sent as a string ``"https://192.0.2.0"``, this will
+            automatically set the scheme to the ``https``
```

Comment:
To be honest, I don't like it 🤔. But I'm not ready to propose a better phrasing for now as well.

Let's just keep it, but I'll get to this later after merging.